src/locales/fr.json

```
@@ -973,9 +986,169 @@
+    "shutdownOrder": "Ordre",
+    "name": "Nom",
+    "group": "Groupe",
+    "loadingPreview": "Chargement de l'aperçu...",
+    "confirmationRequired": "Confirmation Requise",
+    "shutdownConfirmationText": "Cette action va arrêter des serveurs et VMs. Assurez-vous que tous les services sont préparés pour l'arrêt.",
+    "confirmUnderstandConsequences": "Je comprends les conséquences de cet arrêt",
+    "executeShutdown": "Exécuter l'Arrêt",
+    "executing": "Exécution...",
+    "shutdownExecutedSuccessfully": "Arrêt exécuté avec succès",
+    "editGroup": "Modifier le groupe",
+    "updateSuccess": "Groupe mis à jour avec succès",
```

Biome (1.9.4)

[error] The key servers was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)